packages/editor/src/client/routes/pageList.tsx

```diff
@@ -140,13 +146,9 @@ export function PageList() {
             <Cell dataKey="published">
               {(pageRef: PageRefFragment) =>
                 pageRef.published?.publishedAt
-                  ? `${new Date(pageRef.published.publishedAt).toDateString()} ${new Date(
-                      pageRef.published.publishedAt
-                    ).toLocaleTimeString()}`
+                  ? `${dateTimeWithoutSecondsLocalString(new Date(pageRef.published.publishedAt))}`
                   : pageRef.pending?.publishAt
-                  ? `${new Date(pageRef.pending.publishAt).toDateString()} ${new Date(
-                      pageRef.pending.publishAt
-                    ).toLocaleTimeString()}`
+                  ? `${dateTimeWithoutSecondsLocalString(new Date(pageRef.pending?.publishAt))}`
```

Comment:
I recommend to also use the i18next translation service to format the date. Might be a bit more work if it's just the date but it's cleaner this way.